src/provider/persistence/dataLayer/IndexedDb.ts

```diff
@@ -168,19 +168,19 @@ export class IndexedDb<T extends Entity> implements IDatastore<T> {
      * @param {string} tableName The name of the table to perform
      * @param {string} uuid The uuid of the app
      * @public
-     * @returns {Promise<boolean>} A value of whether it was successfully created or not
+     * @returns {Promise<number>} A value of whether it was successfully created or not
      */
-    public removeByUuid(tableName: string, uuid: string): Promise<boolean> {
+    public removeByUuid(tableName: string, uuid: string): Promise<number> {
         return new Promise((resolve, reject) => {
             this.readByUuid(tableName, uuid)
                 .then((result: T[]) => {
                     result.forEach((notification: T) => {
                         this.remove(tableName, notification.id);
                     });
-                    resolve(true);
+                    resolve(result.length);
                 })
                 .catch((err) => {
-                    resolve(false);
+                    resolve(err);
```

Comment:
`reject`?..

Reply:
This is how it was before, and I've been trying my best to leave the storage layer stuff untouched.

Reply:
You're right, there was some weird things here that I'd changed in a potentially bad way.

Fixed now